client/src/config/endpoints.js

```diff
@@ -0,0 +1,9 @@
+const baseUrl = "http://localhost:3300";
+
+const endpoints = {
+  register: `${baseUrl}/api/register`,
+  login: `${baseUrl}/api/login`,
+  jokes: `${baseUrl}/api/jokes`
+};
```

Comment:
An alternative to consider is an [axios instance](https://github.com/axios/axios#creating-an-instance). You can use a base url along with other specifications and then use that instead of the axios from node modules.